CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
```

Comment:
A copy of this file exists at .github/CODE_OF_CONDUCT.md; hence this file should  be removed.